site/docs/education/tutorials/postman-tutorial/overview.md

```diff
@@ -45,24 +45,77 @@ postman files later, you can just create a new environment again to work with
 Select the workspace menu at the top of the Postman interface, and create a new one
 called 'Egeria dojo':
 
-![Creating a new Environment 1](postman-workspace1.png)
-![Creating a new Environment 2](postman-workspace2.png)
-![Creating a new Environment 3](postman-workspace3.png)
+![Creating a new Environment 1](postman-create-workspace1.png)
+![Creating a new Environment 2](postman-create-workspace2.png)
```

Comment:
I wonder if we could use the figure and figcaption https://squidfunk.github.io/mkdocs-material/reference/images/

the png name refers to workspace2 and the last one workspace1 which does not seem to reflect the content
As it implies there are 2 workspaces

Reply:
I've used captions for the screenshots that are not explained in a collapsible section.